geo-types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "geo-types"
-version = "0.6.1"
+version = "0.6.2"
```

Comment:
I *think* this would be considered a minor, non-breaking change as far as semver goes, but I'm willing to be corrected.